### DISCUSSION TITLE
DeltaT Improvements

### DISCUSSION THREAD

**Discussion by worachate001** [Stellarium/stellarium#2338] [Ideas]
As you may have noticed, current value of default deltaT (by Espenak & Meeus 2006) in Stellarium is increasingly different from most recent values published by IERS. The Earth seems to rotate a bit faster in recent years.

I have an idea of using the most recent value and predictions to improve its accuracy that would also help improve predictions of solar eclipses in current year and a few more years in the future.

I would call this "IERS (1974-2032)". The algorithm should have valid range between the years 1974-2032. Before the range, we may use the value from Espenak/Meeus. After this range (2032-), to make the smooth transition, we may use a polynomial equation derived from most recent IERS values and future expected value of the year 2050 (DeltaT = 93).

**Comment by worachate001** [comment]
I forgot to add that we can get the data from here
https://cddis.nasa.gov/archive/products/iers/
(free signup required)

**Comment by alex-w** [comment]
Looks like just another algorithms for DeltaT :)

**Reply by gzotti** [reply]
Of course Espenak/Meeus 2006 cannot be expected to predict accurate values for 2010+. We can simply modify the default solution to use these values after 1974 (and add a note in the SUG).  Or, if IERS has a complete solution (antiquity-near_future), we can add a new one.

**Reply by worachate001** [reply]
I agree to modify it. We have all the necessary data. I will try to do it before the next 0.22.0 release.
